apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx

```diff
@@ -57,53 +64,65 @@ const ConfirmLogin = ({ email, onConfirmLogin }: confirmLoginProps) => {
               <IllusEmailValidation />
             </Stack>
             <Typography variant='h4' pt={8} mb={5} textAlign='center'>
-              {confirmLogin.title}
+              {confirmLogin('title')}
             </Typography>
             {email && (
               <Typography variant='body2' mb={6}>
-                {confirmLogin.body(email)}
+                {confirmLogin('confirmationCodeSent')}
+                <Box component='span' fontWeight='fontWeightMedium'>
+                  {email}
+                </Box>
+                <br />
+                {confirmLogin('confirmationCodeExpires')}
               </Typography>
             )}
             <Typography
               variant='body1'
               sx={{ marginBottom: 1.5, fontWeight: 600 }}
             >
-              {confirmLogin.code}
+              {confirmLogin('code')}
             </Typography>
             <Stack spacing={2} mb={4}>
               <TextField
                 variant='outlined'
                 size='small'
                 onChange={(e) => setCode(e.target.value)}
+                helperText={
+                  codeError
+                    ? confirmLogin('invalidCode')
+                    : emptyCode
+                    ? confirmLogin('emptyCode')
```

Comment:
I think the `emptyCode` key is missing:

![Screenshot 2024-02-09 at 12 07 14](https://github.com/pagopa/developer-portal/assets/4775679/1771fb73-7c11-45e5-b41b-88a16789dfb0)

Reply:
Thanks!